src/plugins/data/public/search/aggs/buckets/date_histogram.ts

```diff
@@ -46,25 +46,21 @@ const updateTimeBuckets = (
 ) => {
   const bounds = agg.params.timeRange ? timefilter.calculateBounds(agg.params.timeRange) : null;
   const buckets = customBuckets || agg.buckets;
-  buckets.setBounds(agg.fieldIsTimeField() && bounds);
+  if (agg.fieldIsTimeField()) {
+    buckets.setBounds(bounds as Bounds);
```

Comment:
Please try to don't use casting to prevent any issues on runtime. Please see line 47 and value (null) which we set in case of no agg.params.timeRange
Probably it cannot bring any issues, but please cleanup that part

Reply:
done